compareTwitterProfiles/index.html

```diff
@@ -0,0 +1,167 @@
+                                    <h4 class="list-group-item-heading">
+                                    <strong>{{userData.followers_count}}</strong> Followers
+                                    </h4>
+                                </div>
+                                <div class="list-group-item">
+                                    <h4 class="list-group-item-heading">
+                                    <strong>{{userData.statuses_count}}</strong> Statuses
+                                    </h4>
+                                </div>
+                                <div class="list-group-item">
+                                    <h4 class="list-group-item-heading">
+                                    <small>{{userData.screen_name}}</small>
+                                    </h4>
+                                </div>
+                            </div>
+                        </div>
+
+                    </div>
+                        <!-- <p>{{userData}}</p> -->
+                    </div>
+              <div class="container">
+                <div class="row">
+                    <div class="col-md-6">
+                        <div id="contactCard">
+
+                        <div class="panel panel-default">
+                            <div class="panel-heading clearfix">
+                                <h3 class="panel-title pull-left">User 2 Profile</h3>
+                            </div>
+                            <div class="list-group">
+                                <div class="list-group-item">
+                                    <img src="{{userData1.profile_image_url}}" alt="" style="pull-left">
+                                    <h4 class="list-group-item-heading">{{userData1.name}}</h4>
+                                </div>
+                                <div class="list-group-item">
+                                    <h4 class="list-group-item-heading">
+                                    <strong>{{userData1.followers_count}}</strong> Followers
+                                    </h4>
+                                </div>
+                                <div class="list-group-item">
+                                    <h4 class="list-group-item-heading">
+                                    <strong>{{userData1.statuses_count}}</strong> Statuses
+                                    </h4>
+                                </div>
+                                <div class="list-group-item">
+                                    <h4 class="list-group-item-heading">
+                                    <small>{{userData1.screen_name}}</small>
+                                    </h4>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </center>
+    <script>
+        var app = angular.module('TweetSearch', []);
```

Comment:
A separate file for Controller would look good. Any reason for adding a controller in the HTML itself?